tests/unit/challenges/test_views.py

```diff
@@ -1587,9 +1587,12 @@ def test_particular_challenge_phase_does_not_exist(self):
             'error': 'ChallengePhase does not exist'
         }
         response = self.client.get(self.url, {})
-        self.assertEqual(response.data, expected)
-        self.assertEqual(response.status_code, status.HTTP_406_NOT_ACCEPTABLE)
-
+        try:
```

Comment:
@rakesh-1601 Can you please refer the issue, this PR is solving?

Reply:
yeah sure sir..

Reply:
https://github.com/Cloud-CV/EvalAI/pull/1753#issuecomment-415982463

Reply:
now the previous error (assertion error) is solved ...but it giving some kind of warning ..plzz review it